src/components/SearchResultDetail/SearchResultDetail.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import * as React from 'react';
+
+import ClipLoader from 'react-spinners/ClipLoader';
+
+import { findSearchResultAction } from '@/app/actions/action';
+import { SearchResultStatus } from '@/constants/enum';
+import toast from '@/lib/toast';
+
+const SearchResultDetail = ({ id }: { id: string }) => {
+  const [searchResult, setSearchResult] = React.useState<any>({});
+  const [loading, setLoading] = React.useState<boolean>(false);
+
+  React.useEffect(() => {
+    async function findSearchResult() {
+      try {
+        setLoading(false);
+        const data = await findSearchResultAction(id);
+        setSearchResult(data);
+      } catch (err: any) {
+        toast(err.message, 'error');
+        setLoading(false);
+      }
+    }
+
+    findSearchResult();
+  }, [id]);
+
+  return (
+    (!loading && (
+      <div className="detail-search-result">
+        <h5 className="detail-search-result__header">
+          Search Result Detail of {searchResult && searchResult.keyword?.toUpperCase()}
+        </h5>
+        <p className="mb-5 text-base text-gray-500 sm:text-lg align-center dark:text-gray-400">
+          Status:{' '}
```

Comment:
Is that to add a space? If yes, why not use string templates?

```suggestion
          Status: {statusText}
```

This would require to create a constant above:

```javascript
const statusText = Object.keys(SearchResultStatus)[
            Object.values(SearchResultStatus).indexOf(searchResult.status)
          ] || 'PENDING'
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/82/commits/6ee44e8d721c2d3583f85799ea9e3f2036ad74f1